mlrun/runtimes/function.py

```diff
@@ -230,6 +232,25 @@ def serving(
 
         return self
 
+    def add_stream_trigger(
```

Comment:
```suggestion
    def add_v3io_stream_trigger(
```
There might be other stream triggers

Reply:
fixed